db/seeds.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
-Dir[Rails.root.join('db/seeds/common/*.rb')].sort.each do |file|
-  puts "Loading file #{file.split(/\/./).last}"
+require 'faker'
+
+Dir[Rails.root.join('db/seeds/common/*.rb')].each do |file|
+  puts "\n Loading file #{file.split(/\//).last}"
```

Comment:
These changes do not seem to belong to fixing mobile ads images, no big deal, but just need to have more concise PR's

Reply:
Thats true! Ill remove them